_drafts/rse-at-egu.md

```diff
@@ -0,0 +1,73 @@
+
+The motivations for RSE national and international activities were further detailed by three representatives from national chapters. 
+David Topping from [UK RSE](http://rse.ac.uk/), the oldest and largest RSE organisation, took a look at the definition of an RSE, at the community history, and it's current state in the UK and beyond.
+Over 15 local groups exists and more are forming at a high rate, even leading to a competition for people.
+Martin Hammitzsch presented the German chapter, [de-RSE](http://de-rse.org/).
+Initiated only 1.5 years ago, he shared the group's objectives, how they work to build a community, challenges they face and some lessons learned: a great resource for the attendees from countries without any organisational structure yet.
+Third up was [Niels Drost](https://www.esciencecenter.nl/profile/dr.-niels-drost) (Netherlands eScience Center), who introduced the youngest European chapter [NL-RSE](http://nl-rse.org/) and its core team, which already generated a considerable reach across the Netherlands.
+
+<blockquote class="twitter-tweet" data-lang="de"><p lang="en" dir="ltr">4 national chapters, their history and plans, are introduced to ~40 participants, albeit meeting at the end of a long conference day. A good start for <a href="https://twitter.com/hashtag/EGU18RSE?src=hash&amp;ref_src=twsrc%5Etfw">#EGU18RSE</a> at <a href="https://twitter.com/hashtag/EGU18?src=hash&amp;ref_src=twsrc%5Etfw">#EGU18</a>. <a href="https://t.co/S2Ob54TSWe">pic.twitter.com/S2Ob54TSWe</a></p>&mdash; Daniel Nüst (@nordholmen) <a href="https://twitter.com/nordholmen/status/984484030031843328?ref_src=twsrc%5Etfw">12. April 2018</a></blockquote>
+<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+After the short talks, we took advantage of the group size, had a complete round of short introductions, and enganged in a discussion.
+We found a big challenge was reaching people engaged in RSEng activities at their work, especially outside of ESSI, and those for whom the "RSE" label does not ring a bell yet.
+The need for outreach also applies to holders of an office within the union, to reach better acknowledgements of the needs of RSEs in universities, scientific unions, and at scientific conferences.
+
+Some good ideas came up and we plan to reach out to EGU and ESSI leadership and share them.
+For example, EGU could increase recognition of RSEs and their work by awarding a medal, by offering a special poster track (not falling under submission limitations) for contributions to scientific software, or by tagging abstracts as "RSE" similarly to the "ECS" badges.
+These are "top-down" ideas that we would like towards within the organisation.
+The potential and usefulness of such domain-specific (taking all EGU members as a group for a moment) actions, who are lateral to the national chapters, was commonly agreed on.
```

Comment:
"potential" is always difficult to use in noun phrases, as it is potentially an adjective ;). Perhaps swap to "usefulness and (overall ?) potential"?